sql-gremlin/src/main/java/org/twilmes/sql/gremlin/adapter/util/SqlGremlinError.java

```diff
@@ -0,0 +1,105 @@
+package org.twilmes.sql.gremlin.adapter.util;
```

Comment:
Needs a copyright.

We have a license check but it is only on our jdbc driver, perhaps we should extend it into sql-gremlin...